lib/chef/knife/core/node_presenter.rb

```diff
@@ -125,7 +125,7 @@ def summarize(data)
             summarized << <<-SUMMARY
 #{key('Recipes:')}     #{Array(node[:recipes]).join(', ')}
 #{key('Platform:')}    #{node[:platform]} #{node[:platform_version]}
-#{key('Tags:')}        #{Array(node[:tags]).join(', ')}
+#{key('Tags:')}        #{node.tags.join(', ')}
```

Comment:
Correct. I fixed all the places where we access node[:tags] which was the only way we'd never initialize it to an Array. Chef::Node#tags makes sure we initialize properly.